### ISSUE TITLE
Can not create new Sandstorm Wekan version, some broken dependency

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#3245]
@ocdtrekkie @zenhack @kentonv 

Any idea what got broken?

I updated some dependencies for Wekan v4.24:
https://github.com/wekan/wekan/blob/master/CHANGELOG.md#v424-2020-08-27-wekan-release

But after I created new Wekan Sandstorm version, that Wekan grain does not start, it shows these errors:

```
** HTTP-BRIDGE: App server exited with status code: 1
** SANDSTORM SUPERVISOR: App exited with status code: 1
** SANDSTORM SUPERVISOR: Starting up grain. Sandbox type: userns
*** Uncaught exception ***
sandstorm/sandstorm-http-bridge.c++:2878: failed: execvp(argvp[0], argvp): No such file or directory; argvp[0] = node
stack: 55694c 6142d3
** HTTP-BRIDGE: App server exited with status code: 1
```

That broken spk file is here: http://releases.wekan.team/broken/

For other Wekan platforms, Wekan Snap does work, and there is not yet any reports that anything else would be broken.

**Comment by zenhack** [comment]
hm, I can't reproduce this; on my local dev setup spinning up a new grain with that .spk works fine. Can you trigger this reliably?

**Comment by xet7** [comment]
Well, my local Sandstorm install is same I used before for testing. It would be nice to know if @ocdtrekkie gets all grains upgraded and working with that spk.

**Comment by xet7** [comment]
Sorry, I found the problem. I did upload wekan-4.24.spk file to my local Sandstorm server using Chrome, so it did not work.

Fix was, use Firefox to upload wekan-4.24.spk to local Sandstorm server. Then everything works.

Also all my old grains did upgrade to newest Sandstorm Wekan version fine.

I will upload that .spk to experimental.

**Comment by ocdtrekkie** [comment]
Shouldn't have an issue uploading an SPK with Chrome... That's actually weirder.
